Formula/plantuml.rb

```diff
@@ -15,12 +18,16 @@ class Plantuml < Formula
     sha256 cellar: :any_skip_relocation, all: "cd5a6877e5b41eacf286bdb9c425730c0a045c9fa4de612a287566da2d9b6415"
   end
 
+  depends_on "ant" => :build # remove when switching to Jar url
   depends_on "graphviz"
   depends_on "openjdk"
 
+  patch :DATA
```

Comment:
This seems to work. Depending on how reliable their `-testdot` is, this may be good to merge.